content/en/community/maintainer_guide/supply-chain-security.md

```diff
@@ -0,0 +1,120 @@
+  ```bash
+  curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | \
+  sh -s -- -b /usr/local/bin v0.54.0
+  chmod +x /usr/local/bin/syft
+  ```
+- Other repositories use JX itself to release different artifacts ([jx-pipeline](https://github.com/jenkins-x-plugins/jx-pipeline) for example). Installing syft is done through this [step](https://github.com/jenkins-x/jx3-pipeline-catalog/blob/4debb1ef44ad846088ce48d7921dd57510b9eda2/tasks/supply-chain-security/task.yaml#L8) in the [jx3-pipeline-catalog](https://github.com/jenkins-x/jx3-pipeline-catalog). To include it, you add this block before the `upload-binaries` step in the [rlease pipeline](https://github.com/jenkins-x-plugins/jx-pipeline/blob/main/.lighthouse/jenkins-x/release.yaml)
+  ```bash
+  - image: uses:jenkins-x/jx3-pipeline-catalog/tasks/supply-chain-security/task.yaml@versionStream
+    name: download-syft
+  ```
+
+### Container images
+
+For container images, We use `syft` to generate SBOMs from the container images (syft is already installed to be used by `goreleaser` and it's not an additional step). After that, we use [oras](https://oras.land/) to push the generated SBOM as an OCI artifact in the same container registry with the container image.
+
+**How to generate and push**
+
+- For JX repositories that uses github actions for the release pipeline ([jx](https://github.com/jenkins-x/jx) for example), we're using oras as the base image for a separate github action. 
+  - Add this block at the end of the release job
+    ```bash
+      - env:
+          GITHUB_TOKEN: ${{ secrets.GIT_BOT_TOKEN }}
+          GIT_USERNAME: jenkins-x
+          DOCKER_REGISTRY_ORG: jenkins-x
+          REPO_NAME: ${{ github.event.repository.name }}
+          VERSION: ${{ steps.prep.outputs.version }}
+        name: Generate and Push SBOM
+        uses: docker://ghcr.io/oras-project/oras:v0.13.0
+        with:
+          entrypoint: .github/workflows/jenkins-x/sbom-container.sh
+    ```
+  - Add this script `.github/workflows/jenkins-x/sbom-container.sh`
+    ```bash
+    #!/bin/sh
+
+    syft ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:latest --scope all-layers -o spdx-json > sbom.json
+    echo $GITHUB_TOKEN | oras push -u $GIT_USERNAME --password-stdin ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:$VERSION-sbom sbom.json
+    echo $GITHUB_TOKEN | oras push -u $GIT_USERNAME --password-stdin ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:latest-sbom sbom.json
+    ```
+
+
+- For repositories that use JX itself to release ([jx-pipeline](https://github.com/jenkins-x-plugins/jx-pipeline) for example):
+
+  - First, include those two steps to use kaniko to build the image and crane to push it (image.tar is required for syft to generate SBOM)
```

Comment:
may be write it differently. Replace the build-push-image step with these two steps.

Reply:
Done